lib/ProductOpener/Tags.pm

```diff
@@ -3539,6 +3541,39 @@ sub add_users_translations_to_taxonomy($) {
 
 }
 
+#Return the last occurence of the property if several are defined in ingredients.txt
+sub find_property_in_ingredient_tree($$$) {
+
+	my $tagtype = shift;
+	my $canon_tagid = shift;
+	my $property = shift;
+
+	while ($canon_tagid) 
+	{
+		if (exists $properties{$tagtype}{$canon_tagid})
+		{
+			foreach(keys $properties{$tagtype}{$canon_tagid})
```

Comment:
no need for the for loop, you can check if $properties{$tagtype}{$canon_tagid}{$property} exists.

Reply:
ok thanks !